barretenberg/cpp/src/barretenberg/proof_system/arithmetization/arithmetization.hpp

```diff
@@ -139,9 +142,16 @@ template <typename FF_> class UltraArith {
 
     struct TraceBlocks {
         UltraTraceBlock pub_inputs;
+        UltraTraceBlock arithmetic;
```

Comment:
I've added all of the blocks that will eventually exist here (and for GUH) but I'm still putting everything in `main` (aside from public inputs and ecc op gates)